compiler/modules/print.llvm/llvm.bal

```diff
@@ -69,6 +69,12 @@ public function constInt(IntType ty, int val) returns ConstValue {
     return new ConstValue(ty, val.toString());
 }
 
+// Corresponds to LLVMConstReal
+public function constReal(RealType ty, float val) returns ConstValue {
+    // XXX do hex conversion
```

Comment:
Just use float:toBitsInt

Reply:
Ahh alright.
By the way, clang is conservative in it's use of hex here, but there shouldn't be any harm in being more precise.

>  The only time hexadecimal floating-point constants are required (and the only time that they are generated by the disassembler) is when a floating-point constant must be emitted but it cannot be represented as a decimal floating-point number in a reasonable number of digits. For example, NaN’s, infinities, and other special values are represented in their IEEE hexadecimal format so that assembly and disassembly do not cause any bits to change in the constants.

Reply:
I created #307 for this.